homeassistant/components/hardware/websocket_api.py

```diff
@@ -34,7 +33,7 @@ async def async_setup(hass: HomeAssistant) -> None:
     """Set up the hardware websocket API."""
     websocket_api.async_register_command(hass, ws_info)
     websocket_api.async_register_command(hass, ws_subscribe_system_status)
-    hass.data[DOMAIN]["system_status"] = SystemStatus(
+    hass.data[DATA_HARDWARE].system_status = SystemStatus(
```

Comment:
If we move this to the package module `async_setup` the attribute doesn't need to be optional.